.github/PROCESS.md

```diff
@@ -6,6 +6,45 @@ We will do all that for you so you can focus on what is really interesting for y
 
 This guide tries to be short and does not mention things that are completely automated by our tooling.
 
+## Team Performance Metrics
+
+1. We want to deliver fast and make our deliverables predictable.
+2. To achieve that, we measure the flow of work and estimate tasks.
+3. The flow of work is measured automatically based on GitHub's pull request workflow.
+4. Tasks are measured manually by engineers during Sprint planning.
+5. Task estimation depends on the following parameters:
+   * Scope.
+     Are the changes required in one or multiple files, packages, components?
+   * Difficulty.
+     How hard is the task?
+     Does it require to cover a lot of test cases?
+     edge cases?
+   * Clarity.
+     Are the description and definition of done clear?
+     Or that's more of a research task?
+6. We use the following T-Shirt Sizes to estimate tasks:
+   * **S**: Small simple clear task.
+   * **M**: Only one parameter can be changed compared to **S** (e.g. a small but not completely clear task).
+   * **L**: Two parameters can be changed compared to **S** (e.g. clear but big and somewhat complex task).
+7. If the team thinks that the task is bigger than **L**, it should be decomposed into smaller tasks.
+8. After every Sprint, we measure how many tasks of each size we were able to complete
+   and discuss what went well and what could be improved.
+   We also look at the flows' metrics to gather more information about the team's dynamic.
+
+## Planning
+
+1. Estimation mistakes should be an exception, not a norm.
```

Comment:
Do you want to add something about retrospectives? Like "If it turns out that a task wasn't estimated correctly, it should be marked with XYZ label and could be discussed at the next retrospective. We offer such discussions to improve our estimation skills"?

Reply:
In general, yes, but that section is about planning. We probably would add one about retro later.